benchmarks/benchmarks/load/__init__.py

```diff
@@ -68,7 +70,7 @@ def time_realise(self, _, __, ___, ____) -> None:
 
 class STASHConstraint:
     # xyz sizes mimic LoadAndRealise to maximise file re-use.
-    params = [[(2, 2, 2), (1280, 960, 5)], ["FF", "PP"]]
+    params = [[(2, 2, 2), (1280, 960, 5), (2, 2, 10000)], ["FF", "PP"]]
```

Comment:
```suggestion
    params = [[(2, 2, 2), (1280, 960, 5), (2, 2, 1000)], ["FF", "PP"]]
```

`10000` takes 20secs per repeat. I tested with `1000` and it's more than enough to demonstrate there's a need for improvement (2secs).

Reply:
Oops, missed updating that one from `10000` to `1000`